modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequestDetailPage.testcase

```diff
@@ -91,6 +91,29 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-181999. Verify that when a MDf is approved, the activity status is Approved and Unclaimed"
+	@priority = 4
+	test ViewActivityStatusWhenMDFIsApproved {
+		task ("When I go to the MDF created") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name 1");
+		}
+
+		task ("And I approved the MDF") {
+			PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+		}
```

Comment:
Hi @marinaasant0s, to approve the MDF, you should log in with the Manager user first